Documentation/check-build.sh

```diff
@@ -21,33 +21,26 @@ cd "${script_dir}"
 mkdir -p "${build_dir}"
 
 build_with_spellchecker() {
-    sphinx-build -b spelling -d "${build_dir}/doctrees" . "${build_dir}/spelling" -q -E 2> "${warnings}"
+    rm -rf "${spelldir}"
+    sphinx-build -b spelling -d "${build_dir}/doctrees" . "${spelldir}" -q -E 2> "${warnings}"
 }
 
 build_with_linkchecker() {
-    sphinx-build -b linkcheck -d "${build_dir}/doctrees" . "${build_dir}/spelling" -q -E
-}
-
-filter_warnings() {
-    grep -v \
-        -e "tabs assets" \
-        -e "misspelled words" \
-        -e "RemovedInSphinx20Warning" \
-        "${warnings}"
```

Comment:
The commit message doesn't seem to explicitly call this out, but does this mean that these previous extraneous warnings are no longer present or no longer trigger a docs build failure? Awesome!

Reply:
Commit log has

```
  [...] Since we touch the script,
  remove filtering expressions which do not seem to catch any existing
  warning.
```

Yes, none of which are present anymore as far as I can tell, so we can just remove this part :)

Reply:
Ah right, I glanced through and missed this. yay :tada: